user-console/src/main/java/org/pentaho/mantle/client/admin/ChangePasswordByUserDialog.java

```diff
@@ -45,6 +47,8 @@ public class ChangePasswordByUserDialog extends GwtDialog implements ServiceCall
   private static final String TEXT_BOX_WIDTH = "260px";
   private static final String SPACER_STYLE_NAME = "spacer";
   private boolean acceptBtnEnabled = false;
+  private final String ALLOWED_CHARS = "^[a-zA-Z0-9_.,:;<>|!@#$%^&*()\\[\\]]+$";
```

Comment:
Can this be static, as Sonar is suggesting? Or will it have an impact when the conversion to JavaScript occurs?
To note that the same may be applicable to the following line (variable 'ALLOWED_CHARS_REGEXP').